packages/notebook/src/actions.tsx

```diff
@@ -1314,6 +1316,81 @@ export namespace NotebookActions {
     Private.handleState(notebook, state, true);
   }
 
+  /**
+   * Render side-by-side.
+   *
+   * @param panel - The current notebook panel.
+   */
+  export function renderSideBySide(
+    panel: NotebookPanel,
+    app?: JupyterLab
+  ): void {
+    Private.sideBySide = true;
+    const applySideBySide = () => {
+      if (!Private.sideBySide) {
+        return;
+      }
+
+      const halfWidth = panel.node.clientWidth / 2;
+      const nodes = document.getElementsByClassName('jp-CodeCell');
```

Comment:
Do not query document if possible (performance). Use `panel.content.widgets` instead.